scripts/apps/authoring/authoring/directives/ReadingTime.js

```diff
@@ -0,0 +1,34 @@
+import * as helpers from 'apps/authoring/authoring/helpers';
+
+ReadingTime.$inject = ['gettextCatalog'];
+export function ReadingTime(gettextCatalog) {
+    return {
+        scope: {
+            item: '=',
+            html: '@'
+        },
+        template: '<span class="char-count reading-time">' +
+            '<span translate> ' + gettextCatalog.getString('{{readingTime}} min read</span></span>'),
```

Comment:
imo there should be either '<span translate>' or 'gettextCatalog', but not both, so either:
```js
'<span class="char-count reading-time" translate>{{readingTime}} min read</span>'
```
or
```js
'<span class="char-count reading-time">'  + gettextCatalog.getString('{{readingTime}} min read') + '</span>'
```